contracts/connectors/reflexer/main.sol

```diff
@@ -0,0 +1,411 @@
+    /**
+     * @dev Open Safe
+     * @param colType Type of Collateral.(eg: 'ETH-A')
+    */
+    function open(string calldata colType) external payable returns (string memory _eventName, bytes memory _eventParam) {
+        bytes32 collateralType = stringToBytes32(colType);
+        require(getCollateralJoinAddress(collateralType) != address(0), "wrong-col-type");
+        uint256 safe = managerContract.openSAFE(collateralType, address(this));
+
+        _eventName = "LogOpen(uint256,bytes32)";
+        _eventParam = abi.encode(safe, collateralType);
+    }
+
+    /**
+     * @dev Close Safe
+     * @param safe Safe ID to close.
+    */
+    function close(uint256 safe) external payable returns (string memory _eventName, bytes memory _eventParam) {
+        uint _safe = getSafe(safe);
+        (bytes32 collateralType, address handler) = getSafeData(_safe);
+        (uint collateral, uint debt) = SafeEngineLike(managerContract.safeEngine()).safes(collateralType, handler);
+
+        require(collateral == 0 && debt == 0, "safe-has-assets");
+        require(managerContract.ownsSAFE(_safe) == address(this), "not-owner");
+
+        managerContract.transferSAFEOwnership(_safe, giveAddr);
+
+        _eventName = "LogClose(uint256,bytes32)";
+        _eventParam = abi.encode(_safe, collateralType);
+    }
+
+    /**
+     * @dev Deposit ETH/ERC20_Token Collateral.
+     * @param safe Safe ID.
+     * @param amt token amount to deposit.
+     * @param getId Get token amount at this ID from `InstaMemory` Contract.
+     * @param setId Set token amount at this ID in `InstaMemory` Contract.
+    */
+    function deposit(
+        uint256 safe,
+        uint256 amt,
+        uint256 getId,
+        uint256 setId
+    ) external payable returns (string memory _eventName, bytes memory _eventParam) {
+        uint _amt = getUint(getId, amt);
+        uint _safe = getSafe(safe);
+        (bytes32 collateralType, address handler) = getSafeData(_safe);
+
+        address colAddr = getCollateralJoinAddress(collateralType);
+        TokenJoinInterface tokenJoinContract = TokenJoinInterface(colAddr);
+        TokenInterface tokenContract = tokenJoinContract.collateral();
+
+        if (isEth(address(tokenContract))) {
+            _amt = _amt == uint(-1) ? address(this).balance : _amt;
+            tokenContract.deposit{value: _amt}();
+        } else {
+            _amt = _amt == uint(-1) ?  tokenContract.balanceOf(address(this)) : _amt;
+        }
+
+        tokenContract.approve(address(colAddr), _amt);
+        tokenJoinContract.join(address(this), _amt);
+
+        SafeEngineLike(managerContract.safeEngine()).modifySAFECollateralization(
+            collateralType,
+            handler,
+            address(this),
+            address(this),
+            toInt(convertTo18(tokenJoinContract.decimals(), _amt)),
+            0
+        );
+
+        setUint(setId, _amt);
+
+        _eventName = "LogDepositCoin(uint256,bytes32,uint256,uint256,uint256)";
```

Comment:
```suggestion
        _eventName = "LogDeposit(uint256,bytes32,uint256,uint256,uint256)";
```
`LogDeposit` event is over here: https://github.com/InstaDApp/dsa-connectors-new/blob/da08f615b5eb97e29594fa6aa8f5286b415e43fa/contracts/connectors/reflexer/events.sol#L7